src/test/java/com/example/java/test/junior/developer/QuestionServiceTest.java

```diff
@@ -0,0 +1,107 @@
+package com.example.java.test.junior.developer;
+
+import com.example.java.test.junior.developer.dto.QuestionDto;
+import com.example.java.test.junior.developer.mapper.QuestionMapper;
+import com.example.java.test.junior.developer.model.Question;
+import com.example.java.test.junior.developer.repository.QuestionRepository;
+import com.example.java.test.junior.developer.service.QuestionService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class QuestionServiceTest {
+
+    @Mock
+    private QuestionMapper questionMapper;
+
+    @Mock
+    private QuestionRepository questionRepository;
+
+    @InjectMocks
+    private QuestionService questionService;
+
+    @Test
+    public void testCreateQuestion() {
+        // Создаем объекты DTO и Entity для тестирования
+        QuestionDto questionDto = new QuestionDto(1, "What is polymorphism?");
+
+        Question questionEntity = new Question();
+        questionEntity.setId(1);
+        questionEntity.setName("What is polymorphism?");
+
+        // Указываем поведение мок-объектов
+
+        when(questionMapper.toEntity(questionDto)).thenReturn(questionEntity);
+        when(questionRepository.save(questionEntity)).thenReturn(questionEntity);
+        when(questionMapper.toDto(questionEntity)).thenReturn(questionDto);
+
+        // Вызываем тестируемый метод
+        QuestionDto result = questionService.createQuestion(questionDto);
+
+        // Проверяем, что методы мок-объектов были вызваны с нужными параметрами
+        verify(questionMapper, times(1)).toEntity(questionDto);
+        verify(questionRepository, times(1)).save(questionEntity);
+        verify(questionMapper, times(1)).toDto(questionEntity);
```

Comment:
No need in this verify block. With new version of Mockito it will fail tests if `when` statements were not executed - https://www.javadoc.io/doc/org.mockito/mockito-core/2.6.5/org/mockito/exceptions/misusing/UnnecessaryStubbingException.html 
You need verify for mock calls that were not stubbed with "when" statement.